orchagent/main.cpp

```diff
@@ -450,6 +451,12 @@ int main(int argc, char **argv)
                 vrf = optarg;
             }
             break;
+        case 'I':
+            if (optarg)
+            {
+                g_heart_beat_interval = atoi(optarg);
```

Comment:
>g_heart_beat_interval [](http://example.com/codeflow?start=16&length=21)

Could you add more error handling?

Reply:
Should we treat 0 as error? an invalid string will also lead to 0.

Reply:
0 means disable heartbeat feature